auditlog/views/auditlog_view.xml

```diff
@@ -0,0 +1,191 @@
+                            </group>
+                            <group colspan="1">
+                                <field name="log_read" invisible="1"/>
+                                <field name="log_write"/>
+                                <field name="log_unlink"/>
+                                <field name="log_create"/>
+                                <!--<field name="log_action"/>-->
+                                <!--<field name="log_workflow"/>-->
+                            </group>
+                        </group>
+                        <!--
+                        <group string="Users">
+                            <field name="user_ids" nolabel="1"/>
+                        </group>
+                        -->
+                    </sheet>
+                </form>
+            </field>
+        </record>
+
+        <record model="ir.ui.view" id="view_auditlog_rule_tree">
+            <field name="name">auditlog.rule.tree</field>
+            <field name="model">auditlog.rule</field>
+            <field name="arch" type="xml">
+                <tree colors="blue:state == 'draft';black:state == 'subscribed'" string="Rules">
+                    <field name="name"/>
+                    <field name="model_id"/>
+                    <field name="log_read"/>
+                    <field name="log_write"/>
+                    <field name="log_unlink"/>
+                    <field name="log_create"/>
+                    <!--<field name="log_action"/>-->
+                    <!--<field name="log_workflow"/>-->
+                    <field name="state"/>
+                </tree>
+            </field>
+        </record>
+
+        <record id="view_auditlog_rule_search" model="ir.ui.view">
+            <field name="name">auditlog.rule.search</field>
+            <field name="model">auditlog.rule</field>
+            <field name="arch" type="xml">
+                <search string="Rules">
+                    <field name="name"/>
+                    <filter domain="[('state','=','draft')]" string="Draft"/>
+                    <filter domain="[('state','=','subscribed')]" string="Subscribed"/>
+                    <field name="model_id"/>
+                    <group expand="0" string="Group By...">
+                        <filter string="State"
+                            domain="[]" context="{'group_by':'state'}"/>
+                    </group>
+                </search>
+            </field>
+        </record>
+
+        <record model="ir.actions.act_window" id="action_auditlog_rule_tree">
+            <field name="name">Rules</field>
+            <field name="res_model">auditlog.rule</field>
+            <field name="type">ir.actions.act_window</field>
+            <field name="view_type">form</field>
+            <field name="view_mode">tree,form</field>
+            <field name="context">{'search_default_draft': 1}</field>
```

Comment:
This search default does nothing, since there is no field or filter named `draft`. My advice is to remove it and use just `{}`.